pkg/platform/docker/docker.go

```diff
@@ -539,7 +539,7 @@ func (idc *Impl) IsContainerRunning(containerName string) (bool, error) {
 	return false, nil
 }
 
-func (idc *Impl) CreateVolume(ctx context.Context, volumeName string, recreate bool) error {
+func (idc *Impl) CreateVolume(ctx context.Context, volumeName string, recreate bool, withOpts bool) error {
```

Comment:
Instead of a bool value, you can give the actual opts, and if there are no options provided, then take the default case.